src/main/kotlin/no/nav/syfo/soknad/SoknadService.kt

```diff
@@ -23,11 +23,31 @@ class SoknadService(
     }
 
     fun handleSykepengesoknad(sykepengesoknad: SykepengesoknadDTO) {
-        if ((sykepengesoknad.status == SoknadsstatusDTO.SENDT && sykepengesoknad.sendtArbeidsgiver != null) &&
-            sykepengesoknad.tom?.isAfter(LocalDate.now().minusMonths(4)) == true
-        ) {
-            soknadDb.insert(sykepengesoknad.toSoknadDbModel())
+        if (shouldHandleSoknad(sykepengesoknad)) {
+            when (sykepengesoknad.status) {
+                SoknadsstatusDTO.NY -> soknadDb.insertOrUpdate(sykepengesoknad.toSoknadDbModel())
+                SoknadsstatusDTO.FREMTIDIG -> soknadDb.insertOrUpdate(sykepengesoknad.toSoknadDbModel())
```

Comment:
Vi trenger kanskje ikke lagre fremtidige søknader? Vi skal aldri vise at de finnes før de foreligger som "NY", og de vil uansett komme tilbake som "NY" når sykmeldingsperioden er over. Tenker på for å minimere data som lagres ;)

Reply:
Det ligger i skissene her https://www.figma.com/file/GZvpBmqBqWVu40FcgBegJP/Valgt-konsept?node-id=1003%3A25650 som planlagte søknader 🤔